reframe/utility/sanity.py

```diff
@@ -521,6 +521,68 @@ def findall(patt, filename, encoding='utf-8'):
     return list(evaluate(x) for x in finditer(patt, filename, encoding))
 
 
+def _callable_name(fn):
+    fn_name = '<unknown>'
+    try:
+        # Assume fn is standard function
+        fn_name = fn.__name__
+    except AttributeError:
+        try:
+            # Assume fn is callable object
+            fn_name = fn.__class__.__name__
+        except AttributeError:
+            pass
+
+    return fn_name
+
+
+def _extractiter_tag(patt, filename, tag, conv, encoding):
+    if isinstance(conv, collections.Iterable):
+        conv = conv[0]
```

Comment:
Should we support that? I don't see the reason. I would rather fail in that case.